jumanji/training/networks/parametric_distribution.py

```diff
@@ -110,9 +110,16 @@ def kl_divergence(
         self, parameters: chex.Array, other_parameters: chex.Array
     ) -> chex.Array:
         """KL divergence is invariant with respect to transformation by the same bijector."""
+        if not isinstance(self._postprocessor, IdentityBijector):
+            raise ValueError("KL is not implemented for non-identity bijectors.")
```

Comment:
I think this is kind of unclear as a user message. Maybe replace with 

"The current post_processor used is a non-identity bijector which does not implement KL"

Reply:
Done